addons/sale_project/views/project_update_template.xml

```diff
@@ -4,12 +4,10 @@
     <template id="sale_project.milestone_deadline_inherit" inherit_id="project.milestone_deadline">
         <xpath expr="//t[contains(@t-if, 'deadline')]" position="before">
             <t t-if="milestone['quantity_percentage']">
-                <span class="text-muted"> (
-                    <t t-if="milestone['sale_line_display_name']">
-                        <t t-out="milestone['sale_line_display_name']"/> -
-                    </t>
-                    <t t-out="100 * milestone['quantity_percentage']"/>% )
-                </span>
+                <t t-if="milestone['sale_line_display_name']">
```

Comment:
I guess the commit `[IMP] sale_project: point 1` is to squash with the first commit, right? If yes, then can you do it?

Thanks in advance! 🙂